docs/integrations/sources/square.md

```diff
@@ -0,0 +1,78 @@
+# Square API
+
+## Overview
+
+The Square Source can sync data from the [Square API](https://developer.squareup.com/reference/square)
+
+Useful links:
+- [Square API Explorer](https://developer.squareup.com/explorer/square)
+- [Square API Docs](https://developer.squareup.com/reference/square)
+- [Square Developer Dashboard](https://developer.squareup.com/apps)
+
+#### Output schema
+
+This Source is capable of syncing the following Streams:
+
+- [Items](https://developer.squareup.com/explorer/square/catalog-api/search-catalog-objects) (Incremental)
+- [Categories](https://developer.squareup.com/explorer/square/catalog-api/search-catalog-objects) (Incremental)
+- [Discounts](https://developer.squareup.com/explorer/square/catalog-api/search-catalog-objects) (Incremental)
+- [Taxes](https://developer.squareup.com/explorer/square/catalog-api/search-catalog-objects) (Incremental)
+- [ModifierLists](https://developer.squareup.com/explorer/square/catalog-api/search-catalog-objects) (Incremental)
+- [Payments](https://developer.squareup.com/reference/square_2021-06-16/payments-api/list-payments) (Incremental)
+- [Refunds](https://developer.squareup.com/reference/square_2021-06-16/refunds-api/list-payment-refunds) (Incremental)
+- [Locations](https://developer.squareup.com/explorer/square/locations-api/list-locations) 
+- [Team Members](https://developer.squareup.com/reference/square_2021-06-16/team-api/search-team-members) (old V1 Employees API) 
+- [List Team Member Wages](https://developer.squareup.com/explorer/square/labor-api/list-team-member-wages)  (old V1 Roles API) 
+- [Customers](https://developer.squareup.com/explorer/square/customers-api/list-customers) 
+- [Shifts](https://developer.squareup.com/reference/square/labor-api/search-shifts) 
+- [Orders](https://developer.squareup.com/reference/square/orders-api/search-orders) 
+
+#### Data type mapping
+
+| Integration Type | Airbyte Type | Notes |
+| :--- | :--- | :--- |
+| `string` | `string` |  |
+| `integer` | `integer` |  |
+| `array` | `array` |  |
+| `object` | `object` |  |
+| `boolean` | `boolean` |  |
+
+#### Features
+
+| Feature | Supported? |
+| :--- | :--- |
+| Full Refresh Sync | Yes |
+| Incremental - Append Sync | Yes |
+| Namespaces | No |
+
+### Requirements
+
+* api_key - The Square API key token 
+* is_sandbox - the switch between sandbox (true) and production (false) environments 
+
+### Setup guide
+
+To get the API key for your square application follow [Geting started](https://developer.squareup.com/docs/get-started) 
+and [Access token](https://developer.squareup.com/docs/build-basics/access-tokens) guides 
+
+## Performance considerations
+
+Some Square API endpoints has different page size limitation   
```

Comment:
The important thing to mention in this section is whether the connector can handle rate limit exceptions

Reply:
Add info to square.md